lib/matplotlib/pyplot.py

```diff
@@ -121,8 +122,7 @@ def install_repl_displayhook():
     Install a repl display hook so that any stale figure are automatically
     redrawn when control is returned to the repl.
 
-    This works with IPython terminals and kernels,
-    as well as vanilla python shells.
+    This works with both with IPython and with vanilla python shells.
```

Comment:
Not a language expert, but this feels doubled. Either "This works with IPython and with vanilla python shells." or "This works with both IPython and vanilla python shells."